src/axom/quest/examples/CMakeLists.txt

```diff
@@ -52,14 +52,18 @@ if(ENABLE_MPI AND MFEM_FOUND AND MFEM_USE_MPI
     if(AXOM_ENABLE_TESTS AND AXOM_DATA_DIR)
         set(_nranks 3)
 
+        # Bugfix: Serialize ordering of DCP tests as workaround for MPI issues 
+        # encountered when these tests ran in parallel
+        set(_dcp_previous_test)
```

Comment:
What exactly was the problem?  `data_collection_util` instances writing to the same filenames?

Reply:
I think the problem is that they're all using the default communicator with custom tags and messages between instances of the distributed closest point example were getting crossed.

I opened #889 to allow users to register a custom communicator, and _\<speculation\>_ I think once we use `MPI_Comm_split` this problem will go away _\</speculation\>_